obspy/core/utcdatetime.py

```diff
@@ -402,11 +403,11 @@ def __init__(self, *args, **kwargs):
                 # year is first (and only) argument
                 year = args[0]
             if not (1 <= int(kwargs['julday']) <= 366) and \
-                    (year % 4 == 0 and (year % 100 != 0 or year % 400 == 0)):
+                    calendar.isleap(year):
```

Comment:
does this fit on the above line now?

Maybe it's a bit shorter and more readable sth like this?

```
days_in_year = calendar.isleap(year) and 365 or 366
if not (....):
    msg = ...
    raise ...
```